internals-js/src/values.ts

```diff
@@ -219,6 +228,35 @@ export function withDefaultValues(value: any, argument: ArgumentDefinition<any>)
 
 const integerStringRegExp = /^-?(?:0|[1-9][0-9]*)$/;
 
+function objectFieldNodeToConst(field: ObjectFieldNode): ConstObjectFieldNode {
+  return { ...field, value: astToConstAST(field.value) };
+}
+
+export function astToConstAST(value: ValueNode): ConstValueNode {
```

Comment:
It might be worth elaborating a bit in a function description that this is really only meant for AST which is already known/expected to be "const" i.e. have no variables in it but just doesn't have the correct typings.

Reply:
Also maybe rename to be a bit more specific, since "ast" can really be any node, but this only accepts and returns `ValueNode`s (or their const equivalent). `valueNodeToConstValueNode`?